pci/src/vfio.rs

```diff
@@ -1000,8 +1060,7 @@ impl VfioCommon {
             0xffff_ffff
         };
 
-        // The config register read comes from the VFIO device itself.
-        self.vfio_wrapper.read_config_dword((reg_idx * 4) as u32) & mask
+        self.read_shadow_dword(((reg_idx * 4) as u32).try_into().unwrap()) & mask
```

Comment:
whiy mutiply *4 here, divide by 4 elsewhere. Unify on `usize` IE an `offset`, and do the math to convert from offset to an index in one place.